src/services/scheduler.ts

```diff
@@ -147,6 +147,71 @@ export async function start(db: IasqlDatabase) {
           }
         }
       },
+      rpc: async (payload: any) => {
+        const { params, opid, moduleName, methodName } = payload;
+        let promise;
```

Comment:
I presume this is where the RPC call is gonna go based on `moduleName`, `methodName`, and `params` above. It won't need to be a `let` variable in here :)

Reply:
yes, I still have to think exactly how it will look but yes, technically it will not be necessary, just looking for the method in the module and calling it.